npm/cypress-schematic/README.md

```diff
@@ -238,17 +267,33 @@ This will create a new spec file named `login.cy.ts` in the default Cypress fold
 Add a `--project=` flag to specify the project:
 
 ```shell script
-ng generate @cypress/schematic:e2e --name=login --project=sandbox
+ng generate @cypress/schematic:spec --name=login --project=sandbox
```

Comment:
a future idea would be to look for the component name matching the `--name` and generating the component there instead of the default. In e2e the default is perfect but in CT we wouldn't really want to generate those there

Reply:
You can currently pass a `--path` argument to save files in a specific location relative to the project root.